tests/modules/nf-core/paftools/sam2paf/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: 'test_paftools_sam2paf:PAFTOOLS_SAM2PAF' {
```

Comment:
Don't really need to set ext.args here to be blank

Reply:
Nearly made it without being prompted for a change, thanks for pointing it out.